### PR TITLE
add 2019.2.11 update

### PR DESCRIPTION
**Pull request by flowerornament**
I'm still confused about how to handle updates in submodules

### CODE REVIEW

File: static-site/posts/updates/2019.2.11-update.udon [hunk 1]
```diff
@@ -0,0 +1,27 @@
+:-  :~  date/'~2019.2.11'
+        title/'~2019.2.11 Update'
+        preview/'A short update from the Tlon team for the week of ~2019.2.11.'
+        author/'~hidrel-fabtel'
+        type/'post'
+    ==
+;>
+
+Here’s a short update from the team for the week of ~2018.2.11
```
Comment (jtobin) [inline static-site/posts/updates/2019.2.11-update.udon]: s/2018/2019

**Comment by msutherl** [comment]
@jtobin thank you sir!

File: static-site/posts/updates/2019.2.11-update.udon (hunk 1 above)
Comment (flowerornament) [reply static-site/posts/updates/2019.2.11-update.udon]: @jtobin thank you sir!

File: static-site/posts/updates/2019.2.11-update.udon [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The past weeks convinced us to try to get an existing `%ames` rewrite running. *Ted* `~rovnys-ricfer` and *Joe* `~master-morzod` got it to compile and are working on documentation and unit tests. *Elliot* `~littel-ponys` is rewriting `%eyre`, our webserver, which is some of the gnarliest code in Arvo. Outbound web requests are now working, which has allowed us to delete old `%eyre` from the [lighter-than-eyre branch](https://github.com/urbit/arvo/tree/lighter-than-eyre), and work continues porting individual `%gall` applications over to `%light`. We’re looking forward to moving Landscape onto new `%eyre` soon. *Logan* `~tacryt-socryp` is rewriting the HTTP client in Vere (Arvo’s interpreter) to better support HTTP/2. This will allow us to provide push notifications in the heretofore rumored Landscape iOS app. Finally, Gavin `ridlur-figbud` and Jimmy `fallyn-balfus` are working on a new design system to enable fast UI prototyping. We’re very excited to share more about that in the near future!
```
Comment (jtobin) [inline static-site/posts/updates/2019.2.11-update.udon:17]: s/ponys/ponnys

File: static-site/posts/updates/2019.2.11-update.udon (hunk 1 above)
Comment (jtobin) [reply static-site/posts/updates/2019.2.11-update.udon]: 🙏

**Pull request closed without merging**